src/main/java/edu/cornell/kfs/coa/batch/dataaccess/impl/ClosedLaborAccountsByDateRangeDaoJdbc.java

```diff
@@ -0,0 +1,74 @@
+package edu.cornell.kfs.coa.batch.dataaccess.impl;
+
+import java.sql.Date;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.text.SimpleDateFormat;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Locale;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.krad.util.ObjectUtils;
+import org.kuali.rice.core.framework.persistence.jdbc.dao.PlatformAwareDaoBaseJdbc;
+import org.springframework.jdbc.core.RowMapper;
+
+import edu.cornell.kfs.coa.batch.CuCoaBatchConstants;
+import edu.cornell.kfs.coa.batch.businessobject.LaborClosedAccount;
+import edu.cornell.kfs.coa.batch.dataaccess.ClosedLaborAccountsByDateRangeDao;
+
+
+public class ClosedLaborAccountsByDateRangeDaoJdbc extends PlatformAwareDaoBaseJdbc implements ClosedLaborAccountsByDateRangeDao {
+    private static final Logger LOG = LogManager.getLogger(ClosedLaborAccountsByDateRangeDaoJdbc.class);
+    
+    private static final String ACCT_FIN_COA_CD = "ACCT_FIN_COA_CD";
+    private static final String ACCT_ACCOUNT_NBR = "ACCT_ACCOUNT_NBR";
+    private static final String SUB_SUB_ACCT_NBR = "SUB_SUB_ACCT_NBR";
+    private static final String ACCT_ACCT_CLOSED_IND = "ACCT_ACCT_CLOSED_IND";
+    private static final String ACCTX_ACCT_CLOSED_DT = "ACCTX_ACCT_CLOSED_DT";
+    
+    @Override
+    public List<LaborClosedAccount> obtainLaborClosedAccountsDataFor(Map<String, Date> dateRange) {
+        List<LaborClosedAccount> closedLaborAccounts = findAllClosedLaborAccountsFor(dateRange);
```

Comment:
Is there a reason to have this function be a one liner, calling the find function?  Why not just have ClosedLaborAccountsByDateRangeDao define findAllClosedLaborAccountsFor and get rid of obtainLaborClosedAccountsDataFor?

Reply:
This class was modeled after another class that needed to perform processing on the data after the "find" before it was returned to the caller.  A reason to leave it as a current one liner is that it would simplify future maintenance since the "find" would already be fully encapsulated.

Reply:
ok